main.cpp

```diff
@@ -175,9 +175,9 @@ int main(int argc, char **argv) {
 	}
 
 	// Build a command line
-	std::string cmdLine;
-	for (int i = 1; i < argc; i++) {
-		if (i != 1) cmdLine += ' ';
+	std::string cmdLine = "\"" + files::pathToWindows(std::filesystem::absolute(argv[1])) + "\"";
```

Comment:
can you apply the quoting logic below for the first arg too, just with `if (i == 1) arg = files::pathToWindows(std::filesystem::absolute(arg));`?